src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,124 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column()
+    private String genre;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column(nullable = false)
+    private String image;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String subtitle;
+
+    @Column(nullable = false)
+    private String publisher;
+
+    @Column(nullable = false)
+    private String year;
+
+    @Column(nullable = false)
+    private Integer pages;
+
+    @Column(nullable = false)
+    private String isbn;
+
+    public Book(String author, String image, String title, String subtitle,
```

Comment:
You need to add a constructor with no args too. JPA use it to create the instance object.